src/Interfaces.java

```diff
@@ -0,0 +1,122 @@
+Interface 1: Kernel Interface
```

Comment:
I get the idea here, but it's meant to be a literal interface. Something like the following:

```java
public interface BookshelfKernel extends Standard<Bookshelf> {
    ...
}
```